core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.directive.ts

```diff
@@ -454,9 +454,7 @@ angular.module('oppia').directive('stateInteractionEditor', [
             $scope.customizationModalReopened = false;
 
             $scope.userBlueImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/avatar/user_blue_72px.png');
-            $scope.userBlackImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/avatar/user_black_72px.png');
+              '/avatar/user_blue_72px.webp');
```

Comment:
Could we support both PNG and WebP here?

Reply:
@darksun27 What about this?

Reply:
@darksun27 ping!